index.js

```diff
@@ -4,63 +4,74 @@ let klawSync = require('klaw-sync');
 let cheerio = require('cheerio');
 
 function findMatchingFile (pattern, bundle) {
-    for (let file in bundle) {
-        if (pattern.test(file)) {
-            return file;
-        }
-    }
+	for (let file in bundle) {
+		if (pattern.test(file)) {
+			return file;
+		}
+	}
+}
+
+function normalizedFile(file) {
+	return `${process.cwd()}/${file.match(/[^\.\/](.+)/g)}`
```

Comment:
This ignores leading `.` and/or `/` with extreme prejudice. The thinking being that any `exclude` must necessarily reside within an `include`, so strict normalization is the sanest approach.